sktime/transformations/base.py

```diff
@@ -617,6 +869,45 @@ def _transform(self, X, y=None):
         """
         raise NotImplementedError("abstract method")
 
+    def _inverse_transform(self, X, y=None):
+        """Inverse transform X and return an inverse transformed version.
+
+        core logic
+
+        Parameters
+        ----------
+        X : Series or Panel of mtype X_inner_mtype
+            if X_inner_mtype is list, _inverse_transform must support all types in it
+            Data to be transformed
+        y : Series or Panel, default=None
+            Additional data, e.g., labels for transformation
+
+        Returns
+        -------
+        inverse transformed version of X
```

Comment:
What is the type of the return?

Reply:
I added detail and a reference to the extension templates